src/components/search-results/index.js

```diff
@@ -243,7 +243,7 @@ class SearchResults extends Component {
     const gridStyle = searchComplete && searchItems.length === 0 ? hideGridStyle : showGridStyle;
     return (
       <div className='gridContainer'>
-        {!isInitialTag && searchComplete && hotelItems.length === 0 ? noHotelsErrorMessage : ''}
+        {!isInitialTag && ranking && searchComplete && hotelItems.length === 0 ? noHotelsErrorMessage : ''}
```

Comment:
@lennym in light of your most recent PR https://github.com/numo-labs/isearch-ui/pull/521 can we _safely_ know when there are no hotels available?